core/src/tangram.cpp

```diff
@@ -229,7 +229,7 @@ void applySceneUpdates() {
 
```

Comment:
I am not sure I get your comment here @hjanetzek about m_nextScene. Copying of m_nextScene happens on the same thread so it should be in sync.